### DISCUSSION TITLE
For mariadb,can't not set column as primary key

### DISCUSSION THREAD

**Discussion by AlexLuya** [dbeaver/dbeaver#589] [Q&A] [answered]
![workspace 1_021](https://cloud.githubusercontent.com/assets/303353/16722992/db41fe56-477c-11e6-981e-7320d37cc8b2.png)

**Comment by serge-rider** [comment, marked as answer]
This is a read-only property (shows key type).
To make some column a key you need to create a constraint (primary key) or an index.

**Comment by serge-rider** [comment]
This is a read-only property (shows key type).
To make some column a key you need to create a constraint (primary key) or an index.
